<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="exitLabel">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>4</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>âœ•</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="windowName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="text">
     <string>Gas Station</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="startPage">
     <attribute name="title">
      <string>Start</string>
     </attribute>
     <widget class="QLabel" name="label_T">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>T    =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_M">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>M   =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Z">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Z    =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_P">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>P    =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Q1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Q1 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_V">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>V   =</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>761</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFEED9</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans CJK JP weight=250'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Description:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;T - Simulation time [minutes]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;M - Time scale (1 unit of real time : M units of simulation time) [1]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Z - Flow intensity [cars/minute]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;P - Probability of choosing the first type of fuel [0; 1]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Q1 and Q2 - Upper and lower bounds of uniform distribution of required fuel amount [liters]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;V - Refueling speed [liters/minute]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Q2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Q2 =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_T">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_M">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_Z">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_P">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_Q1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_Q2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="value_V">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startSimulationButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>191</width>
        <height>56</height>
       </rect>
      </property>
      <property name="text">
       <string>Start simulation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseSimulationButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>191</width>
        <height>56</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause simulation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopSimulationButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>191</width>
        <height>56</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop simulation</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="simulationProgress">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>591</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelProgress">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>67</y>
        <width>74</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="graphicalViewPage">
     <attribute name="title">
      <string>Graphical view</string>
     </attribute>
     <widget class="QLabel" name="place_1_0">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_2_1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gas_0">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_0_0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_2_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>240</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_0_1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_1_1">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="place_2_0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gas_1">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>111</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../gas_station.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_0_0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_0_1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_1_0">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_1_1">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="statisticPage">
     <attribute name="title">
      <string>Statistic</string>
     </attribute>
     <widget class="QLabel" name="meanLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mean car arrival time [seconds]:</string>
      </property>
     </widget>
     <widget class="QLabel" name="meanCarArrivalTimeLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="randomTestPage">
     <attribute name="title">
      <string>Random test</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Uniform distribution</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Exponential distribution</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uniformRandomSampleLength">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>70</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Sample length</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="exponentialRandomSampleLength">
      <property name="geometry">
       <rect>
        <x>512</x>
        <y>70</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Sample length</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateRandomUniformButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateRandomExponentialButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>110</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Expectation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>230</y>
        <width>151</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Standard deviation</string>
      </property>
     </widget>
     <widget class="QLabel" name="expectationUniformLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="stdUniformLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="expectationExponentialLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="stdExponentialLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>windowName</zorder>
   <zorder>exitLabel</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
